.github/workflows/ci.yml

```
@@ -42,8 +42,8 @@ jobs:
       - name: Typecheck
         run: yarn types:check
 
-  lint:
-    name: Lint
+  lint_and_format:
+    name: Lint & Format
```

I consider formatting here just as a kind of linting so i think that "lint" is still OK here and if we ever add a new lint type I'd rather like to avoid renaming this to ``lint_and_format_and_xxx` ;p
```suggestion
    name: Lint
```